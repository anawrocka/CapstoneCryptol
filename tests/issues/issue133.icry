:load issue133.cry
:exhaust \x -> gcd 0 x == x
:prove \x -> gcd 0 x == x
