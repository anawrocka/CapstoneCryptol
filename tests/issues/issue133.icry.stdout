Loading module Cryptol
Loading module Cryptol
Loading module Main
Using exhaustive testing.
passed 256 tests.
Q.E.D.
Q.E.D.
